node/test/util/status.js

```diff
@@ -900,7 +935,11 @@ describe("Status", function () {
         const cases = {
             "trivial": {
                 state: "x=S",
-                regex: /On branch.*master.*\nnothing to commit.*/,
+                regex: /On branch.*master.*\n.*nothing to commit.*/,
+            },
+            "rebase": {
+                state: "x=S:C2-1;C3-1;Bfoo=3;Bmaster=2;Efoo,2,3",
+                regex: /rebas/,
```

Comment:
Really needs to reference commit ids as well; I've got it on my todo list to make all those regexes a little more specific.